docs/config_entries_options_flow_handler.md

```diff
@@ -39,15 +40,51 @@ class OptionsFlowHandler(config_entries.OptionsFlow):
 
         return self.async_show_form(
             step_id="init",
-            data_schema=vol.Schema(
-                {
-                    vol.Required(
-                        "show_things",
-                        default=self.config_entry.options.get("show_things"),
-                    ): bool
-                }
+            data_schema=self.add_suggested_values_to_schema(
```

Comment:
On line 39 perhaps take the opportunity to remove `title` within `async_create_entry`

Reply:
> On line 39 perhaps take the opportunity to remove `title` within `async_create_entry`

done